From 6521762e0fb2c41df249aeca527a4419a2631867 Mon Sep 17 00:00:00 2001
From: Daniel Golle <daniel@makrotopia.org>
Date: Tue, 6 Feb 2024 03:18:21 +0000
Subject: [PATCH] adb: fix build on glibc systems

Recent changes to adb_walk_adb.c added use of 'PRIu64' which requires
on glibc systems.

Fixes: c5d8d28 ("adb: implement ADB_BLOCK_EXT flavor for large files")
---
 src/adb_walk_adb.c | 1 +
 1 file changed, 1 insertion(+)

--- a/src/adb_walk_adb.c
+++ b/src/adb_walk_adb.c
@@ -2,6 +2,7 @@
 
 #include <stdio.h>
 #include <unistd.h>
+#include <inttypes.h>
 #include "apk_adb.h"
 #include "apk_applet.h"
 #include "apk_print.h"
